File: src/test/java/org/cerion/stocklist/PriceListTest.java
package org.cerion.stocklist;

import static org.junit.Assert.*;

import org.junit.BeforeClass;
import org.junit.Test;

public class PriceListTest {

	static PriceList mPriceList;

	@BeforeClass
	public static void init() {

		mPriceList = Helper.getSP500TestData();
	}

	@Test
	public void averageYearlyGain() {
		float gain = mPriceList.averageYearlyGain();

		assertEquals(0.0215, gain, 0.0001);
	}


}

File: src/test/kotlin/PriceListTest.kt
package org.cerion.stocklist

import org.junit.Assert.assertEquals
import org.junit.Test

class PriceListTest : TestBase() {

    @Test
    fun averageYearlyGain() {
        val gain = priceList.averageYearlyGain()
        assertEquals(0.0215, gain.toDouble(), 0.0001)
    }
}
